qpid-docker/docker-build.sh

```diff
@@ -0,0 +1,175 @@
+#!/bin/sh
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from_local_dist=
+from_release=
+local_dist_path=
+qpid_version=
+MY_NAME=$(basename "$0")
+
+print_help()
+{
+  cat << END_OF_HELP
+Usage: $MY_NAME [OPTION]...
+
+ options:
+
+  --local-dist-path  Path to the local Apache Qpid Broker-J distribution
+  --qpid-version     Apache Qpid Broker-J version
```

Comment:
The release based option should go first, its the normal use case. The description could be clearer, something like "Apache Qpid Broker-J release version to build with" and "Path to the local Apache Qpid Broker-J distribution to build with"

Using "release" within the actual option name for release-version builds rather than simply "qpid-version" would also help make it more obvious that its a choice of 'release vs local-build' distribution.